litellm/tests/test_notdiamond_completion.py

```diff
@@ -0,0 +1,162 @@
+nd_tools_model_list = [{'provider': 'openai', 'model': 'gpt-3.5-turbo'}, {'provider': 'openai', 'model': 'gpt-3.5-turbo-0125'}, {'provider': 'openai', 'model': 'gpt-4'}, {'provider': 'openai', 'model': 'gpt-4-0613'}, {'provider': 'openai', 'model': 'gpt-4o'}, {'provider': 'openai', 'model': 'gpt-4o-2024-05-13'}, {'provider': 'openai', 'model': 'gpt-4-turbo'}, {'provider': 'openai', 'model': 'gpt-4-turbo-2024-04-09'}, {'provider': 'openai', 'model': 'gpt-4-turbo-preview'}, {'provider': 'openai', 'model': 'gpt-4-0125-preview'}, {'provider': 'openai', 'model': 'gpt-4-1106-preview'}, {'provider': 'anthropic', 'model': 'claude-3-opus-20240229'}, {'provider': 'anthropic', 'model': 'claude-3-sonnet-20240229'}, {'provider': 'anthropic', 'model': 'claude-3-5-sonnet-20240620'}, {'provider': 'anthropic', 'model': 'claude-3-haiku-20240307'}, {'provider': 'mistral', 'model': 'mistral-large-latest'}, {'provider': 'mistral', 'model': 'mistral-small-latest'}, {'provider': 'cohere', 'model': 'command-r'}, {'provider': 'cohere', 'model': 'command-r-plus'}, {'provider': 'google', 'model': 'gemini-pro'}, {'provider': 'google', 'model': 'gemini-1.5-pro-latest'}, {'provider': 'google', 'model': 'gemini-1.5-flash-latest'}, {'provider': 'google', 'model': 'gemini-1.0-pro-latest'}]
+
+
+def test_completion_notdiamond():
+    try:
+        litellm.set_verbose = False
+        messages = [
+            {
+                "role": "user",
+                "content": "Hey",
+            },
+        ]
+        for model in nd_model_list:
+            print("model:", f"{model['provider']}/{model['model']}")
```

Comment:
As a general rule, published libraries should avoid `print` statements outside of scripts. The maintainers will also ask you to remove it.

Reply:
Missed these print statements, thanks for pointing it out!